mcp-servers/oracle-sqlcl-pvc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
is there any reason why we have to do it as k8 object rather than helm?

Reply:
This was a lingering file, no longer needed and has been deleted.